app/src/main/java/com/globant/practice/presentation/view/adapter/SubscriberDetailsAdapter.java

```diff
@@ -0,0 +1,215 @@
+     */
+    @Override
+    public int getItemCount() {
+        return subscriberRepositories == null ? 0 : subscriberRepositories.size() + 1;
+    }
+
+    /**
+     * Returns the ViewType depending of the position of the view
+     *
+     * @param position position of the view
+     * @return 0 for the subscriber profile item and 1 for the subscriber repository item
+     */
+    @Override
+    public int getItemViewType(int position) {
+        return position == INDEX_ZERO ? SUBSCRIBER_PROFILE_ITEM : SUBSCRIBER_REPOSITORY_ITEM;
+    }
+
+    /**
+     * Inner class that manages the items of the view
+     */
+    public class SubscriberDetailsViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+        private TextView name;
+        private TextView location;
+        private TextView company;
+        private TextView followers;
+        private TextView following;
+        private TextView publicRepos;
+        private ImageView avatar;
+        private Repository repository;
+
+        /**
+         * Construct method of the SubscriberDetailsViewHolder, initializes the view items depending of
+         * the tag of the view
+         *
+         * @param itemView view item
+         */
+        public SubscriberDetailsViewHolder(View itemView) {
+            super(itemView);
+            if (itemView.getTag().equals(context.getString(R.string.subscriber_details_profile_layout_tag))) {
+                avatar = (ImageView) itemView.findViewById(R.id.subscriber_profile_avatar);
+                name = (TextView) itemView.findViewById(R.id.subscriber_details_name);
+                location = (TextView) itemView.findViewById(R.id.subscriber_details_location);
+                company = (TextView) itemView.findViewById(R.id.subscriber_details_company);
+                followers = (TextView) itemView.findViewById(R.id.subscriber_details_followers);
+                following = (TextView) itemView.findViewById(R.id.subscriber_details_following);
+                publicRepos = (TextView) itemView.findViewById(R.id.subscriber_details_publicrepos);
+            } else {
+                name = (TextView) itemView.findViewById(R.id.subscriber_details_repo_name);
+                itemView.setOnClickListener(this);
+            }
+        }
+
+        /**
+         * Sets the subscriber profile data on the view items
+         */
+        public void blindSubscriberDetails() {
+            Glide.with(context).load(profile.getAvatarUrl()).centerCrop()
+                    .placeholder(R.drawable.ic_account_circle_black_24dp).crossFade().into(avatar);
+            if (TextUtils.isEmpty(profile.getName())) {
+                name.setText(profile.getLogin());
+            } else {
+                name.setText(profile.getName());
+            }
+            location.setText(String.format("%s %s", context.getString(R.string.subscriber_details_location), profile.getLocation() != null ? profile.getLocation() : ""));
```

Comment:
Define a String resource that has this format. Example:
`<string name="test">Label: %$s<string>`